src/Couchbase.Lite.Tests.Shared/URLEndpointListenerTest.cs

```diff
@@ -634,7 +672,7 @@ public void TestReplicatorAndListenerOnSameDatabase()
 
             repl1.Start();
             repl2.Start();
-            WaitHandle.WaitAll(new[] {wait1.WaitHandle, wait2.WaitHandle}, _timeout)
+            WaitHandle.WaitAll(new[] {wait1.WaitHandle, wait2.WaitHandle}, TimeSpan.FromSeconds(20))
```

Comment:
The point of assigning `_timeout` is to make things dynamic, just FYI

Reply:
yeah, the default value is 15 sec... Just want to be safe to wait longer for two separate replicators to stop.